assets/css/main.css

```diff
@@ -1,624 +1,693 @@
+/*
+Red/Teal
+rgba(204, 40, 41, 1), #4ba58c
+*/
+/*
+Grey-Blue
+
+*/
 html, body {
   margin: 0;
   padding: 0;
   height: 100%;
-  font-family: helvetica, sans-serif;
+  font-family: 	"mr-eaves-xl-modern", helvetica;
+  font-weight:400;
   line-height: 1.3;
-  color: #2b2b2b;
+  color:rgba(37,52,62, 1);
   background: #FBFCFC;
   font-size: 110%;
 }
 
-a {
-  color: #F1853D;
-  text-decoration: none;
-}
-
-button {
-  background: #F1853D;
-  border: 0;
-  color: #fff;
-  padding: 13px 30px;
-  font-size: 1.35em;
-}
-
-input,textarea {
-  font-size: inherit;
-  font-family: inherit;
-  -webkit-appearance: none;
-  border-radius: 0;
-}
-
-fieldset {
-  border: 0;
-  padding: 0;
-  margin: 0;
-}
-
-fieldset[disabled], button[disabled] {
-  opacity: 0.5;
-  filter: alpha(opacity=50);
+div{
+  display:inline-block;
+  width:100%;
```

Comment:
This might now be a good idea to overwrite default div styling. Divs (and spans) are used generally as the go to elements for plugins and scripts because they don't have any styling. If you want to have default styling on divs I might suggest a generic classname like `.yi-component` or something. Then you could do like `<div class="yi-component">` and you wont have to worry about weird styling from 3rd party scripts.